components/MediaCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <section>
+    <h3 class="py-12 capitalize">Latest {{ mediaType }}</h3>
+    <article
+      class="items-center grid grid-cols-1 pb-12 lg:gap-12 lg:grid-cols-9"
+    >
+      <div class="lg:col-span-5">
+        <h4 class="text-xl lg:mb-4">
+          {{ details.title }}
+        </h4>
+        <p>
+          {{ formatDateWithDots(details.date || details.pubDate) }}
+        </p>
+        <p>
+          {{ details.description }}
+        </p>
+      </div>
+      <nuxt-img
+        class="order-first h-auto m-auto max-w-md w-full pb-10 lg:col-span-2"
```

Comment:
I'm curious about the use of `order-first` here. Since we don't seem to be changing the ordering based on screen size or anything, could we just actually move this element to be the first element instead?

Reply:
For accessibility - header comes before other elements in any section

Reply:
Ahhh, gotcha, that seems fine then

Reply:
That's a great tip, I never thought of it that way!